lnrpc/websocket_proxy.go

```diff
@@ -230,6 +230,12 @@ func (p *WebsocketProxy) upgradeToWebSocketProxy(w http.ResponseWriter,
 	// Ping write loop: Send a ping message regularly if ping/pong is
 	// enabled.
 	if p.pingPongEnabled() {
+		// We'll send out our first ping in pingInterval. So the initial
+		// deadline is that interval plus the time we allow for a
+		// response to be sent.
+		initialDeadline := time.Now().Add(p.pingInterval + p.pongWait)
```

Comment:
Nit: looks like these changes should've been squashed on commit https://github.com/lightningnetwork/lnd/pull/5237/commits/de175da7bd17ac3ba9e5cc8d8251039499d7dc82 instead?

Reply:
Oops, yes definitely. Fixed.